functions/src/glean.ts

```diff
@@ -4,8 +4,14 @@ import axios from "axios";
 import functions from "firebase-functions";
 import { Mutex, Semaphore, withTimeout } from "async-mutex";
 import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
 
-const RWP_pkg = JSON.parse(fs.readFileSync("package.json").toString());
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const RWP_pkg_file = fs.readFileSync(
+  path.resolve(__dirname, "../../package.json")
```

Comment:
Maybe we should use the `package.json` version from the `functions/` dir? I think we might want to separate deploy of functions and the web site at some point anyway.